### DISCUSSION TITLE
Adding Machines from Pcchips,Asus,Gigabyte that have onboard Video

### DISCUSSION THREAD

**Discussion by TechGuyC** [86Box/86Box#3838] [General]
besides the Socket 7,there is no Super socket 7,Slot 1,Socket 370 motherboards from asus,pcchips,gigabyte with onboard video

**Comment by ghost** [comment]
So, what are you talking about in this conversation, then what is this? Are you adding some machines like ASUS, PC Chips, Gigabyte, etc?
Or are you just asking about the onboard video in these 3 motherboard brands?

**Reply by TechGuyC** [reply]
Asking about onboard video on these 3 motherboard brands

**Comment by ghost** [comment]
So, you told me in the Discord 86Box Server that you're going to add these SiS 5598 boards with a SiS 6326 internal video controller?
![image](https://github.com/86Box/86Box/assets/147709293/a263ca6f-e1a7-4ad7-86eb-d02bd826e169)
And also, in the image, what are you talking about, like: ***there is socket 7 motherboards with onboard video,but they are from OEM brands (Compaq,Toshiba),But None from Asus or Pcchips***
These are not Compaq and Toshiba OEM's, they're look like retail versions of these motherboards, and the Toshiba ones are from Intel Socket 7 boards only.

**Comment by TechGuyC** [comment]
> So, you told me in the Discord 86Box Server that you're going to add these SiS 5598 boards with a SiS 6326 internal video controller? 

No
>there is socket 7 motherboards with onboard video,but they are from OEM brands (Compaq,Toshiba),But None from Asus or Pcchips**_ These are not Compaq and Toshiba OEM's, they're look like retail versions of these motherboards, and the Toshiba ones are from Intel Socket 7 boards only.

And i wanted motherboards of wich were used in generic or OEM AT or ATX gabinets (Asus,Pcchips,Gigabyte)

**Reply by ghost** [reply]
Then, is this your Discord username (Felipe)? And why did you left the Discord 86Box server while I was asleep?

**Comment by TechGuyC** [comment]
> Then, is this your Discord username (Felipe)? And why did you left the Discord 86Box server while I was asleep?

Yes,because i did not want to stay in the server

**Reply by ghost** [reply]
So, are you really sure you're gonna add these SiS 5598 with a SiS 6326 internal video controller machines here:
![image](https://github.com/86Box/86Box/assets/147709293/e3ac7a09-ffde-48f2-ac66-4f492d1e9b27)
We didn't emulate that SiS chipset, because of the on board video controller.

**Reply by ghost** [reply]
Most of these boards, like the ASUS SP97-XV, ASUS SPAX-M (HP OEM), PCChips M571/M596 both uses the SiS 6326 internal video controller, and the PCChips M571/M596 chipset has a PCChips TX Pro II, which is an OEM version of the SiS 5597/5598.

**Comment by TechGuyC** [comment]
> So, are you really sure you're gonna add these SiS 5598 with a SiS 6326 internal video controller machines here: We didn't emulate that SiS chipset, because of the on board video controller.

I dont know how to add chipsets or motherboards in emulator

about the SIS video,i also asked in the discussions to add the SIS 6326 AGP and PCI graphics card (the 1st documentation said only about the AGP and was incomplete) then found another documentation that said about PCI and AGP,and have sended that other documentation,but github did not say anything about progress or tasks or pull requests

heres is the 2nd documentation if you want to know
[6326.documentation.pdf](https://github.com/86Box/86Box/files/13464244/6326.documentation.pdf)

**Comment by OBattler** [comment]
The reason why is simple: accelerated graphics cards are not easy to emulate. They require a lot of time. Right now, we have plenty of other stuff on our TODO list so why I agree these would be nice to have, they are not currently high priority.

**Comment by OBattler** [comment]
And guestuser0001, I'm very sorry but you're just wasting TechGuyC's time with useless questions. Please refrain from doing that.

**Reply by ghost** [reply]
Okay, indeed!
